lib/puppet/ffi/windows/constants.rb

```diff
@@ -0,0 +1,301 @@
+require 'ffi'
+
+module Puppet::FFI::Windows
+  module Constants
+    extend FFI::Library
```

Comment:
do we need this line, since we're only setting constants?

Reply:
I think the `enum(` function is using it in https://github.com/puppetlabs/puppet/pull/8275/files#diff-b4cf83b879a6e914d920d8b280de46c2R60